packages/now-static-build/test/fixtures/53-native-gems/gem-install-rmagick.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -euo pipefail
+
+mkdir dist
```

Comment:
`-p` won't fail if the dir already exists.

```suggestion
mkdir -p dist
```